tests/schema/schema.go

```diff
@@ -0,0 +1,22 @@
+package schema
+
+import (
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+type EndpointStatus struct {
+	Endpoint string
+	Status   *clientv3.StatusResponse
+}
+
+type EndpointHashKV struct {
+	Endpoint string
+	HashKV   *clientv3.HashKVResponse
+}
+
+type EndpointHealth struct {
```

Comment:
On the other hand this struct looks like it should be reused. Have you checked if there is already similar struct defined to  marshall json?

Reply:
Just like follow, it's private struct.  Can I change then to Public? 
https://github.com/kkkkun/etcd/blob/00c6090110ba053ec4f521702ff6e95ff61352eb/etcdctl/ctlv3/command/ep_command.go#L180
https://github.com/kkkkun/etcd/blob/00c6090110ba053ec4f521702ff6e95ff61352eb/etcdctl/ctlv3/command/ep_command.go#L209

Reply:
I think the decision should be based whether struct is part of client/server API or an internal part of binary.

 In case of two you linked `epHashKV` and `epStatus` those are part of `etcdctl` binary as so I don't think they should be public. As proposed in comment above I think returning just response `*clientv3.StatusResponse` and `*clientv3.HashKVResponse` is enough. Client doesn't distinquish between endpoints so it doesn't matter which response is from which endpoint.
 
 As for EndpointHealth, I assumed that endpoints returns some JSON generated by some struct. I would look into how the response is parsed in client.

Reply:
> 

That's make sense. I will fix.